docs/cedarling/cedarling-authz.md

```diff
@@ -9,49 +9,49 @@ tags:
 
-The JWTs, Resource, Action, and Context are sent in the authz request. Cedar Pricipals entities 
-are derived from JWT tokens. The OpenID Connect ("OIDC") JWTs are joined by the Cedarling to create 
-User and Role entities; the OAuth access token is used to create a Workload entity, which is the 
+The JWTs, Resource, Action, and Context are sent in the authz request. Cedar Pricipals entities
+are derived from JWT tokens. The OpenID Connect ("OIDC") JWTs are joined by the Cedarling to create
+User and Role entities; the OAuth access token is used to create a Workload entity, which is the
 software that is acting on behalf of the Person (or autonomously). The Cedarling validates that
 given its policies, both the Person and Workload are authorized.
 
 The Cedarling maps "Roles" out-of-the-box. In Cedar, Roles are a special kind of Principal. Instead
-of saying "User can perform action", we can say "Role can perform action"--a convenient way to 
+of saying "User can perform action", we can say "Role can perform action"--a convenient way to
 implement RBAC. Developers can specify which JWT claim is used to map Cedar Roles. For example, one
-domain may use the `role` user claim of the OpenID Userinfo token; another domain may use the 
+domain may use the `role` user claim of the OpenID Userinfo token; another domain may use the
 `memberOf` claim in the OIDC id_token.
 
-Developers can also express a variety of policies beyond the limitations of RBAC by expressing ABAC 
-conditions, or combining ABAC and RBAC conditions. For example, a policy like Admins can access a 
-"private" Resource from the private network, during business hours. In this case "Admins" is the role, 
-but the other conditions are ABAC. Policy evaluation is fast because Cedar uses the RBAC role to 
+Developers can also express a variety of policies beyond the limitations of RBAC by expressing ABAC
+conditions, or combining ABAC and RBAC conditions. For example, a policy like Admins can access a
+"private" Resource from the private network, during business hours. In this case "Admins" is the role,
+but the other conditions are ABAC. Policy evaluation is fast because Cedar uses the RBAC role to
 "slice" the data, minimizing the number of entries on which to evaluate the ABAC conditions.
 
-The OIDC id_token JWT represents a Person authentication event. The access token JWT represents a 
-Workload authentication event. These tokens contain other interesting contextual data. The id_token 
-tells you who authenticated, when they authenticated, how they authenticatated, and optionally other 
-claims like the User's roles. An OAuth access token can tell you information about the Workload that 
-obtained the JWT, its extent of access as defined by the OAuth Authorization Server (*i.e.* the 
+The OIDC id_token JWT represents a Person authentication event. The access token JWT represents a
+Workload authentication event. These tokens contain other interesting contextual data. The id_token
```

Comment:
for consistency, let's format all the keywords like `id_token` as code
```suggestion
The OIDC `id_token` JWT represents a Person authentication event. The access token JWT represents a
Workload authentication event. These tokens contain other interesting contextual data. The `id_token`
```

Reply:
resolved